apps/docs-snippets/src/guide/wallets/checking-balances.test.ts

```diff
@@ -0,0 +1,61 @@
+import { generateTestWallet } from '@fuel-ts/account/test-utils';
+import { ASSET_A } from '@fuel-ts/utils/test-utils';
+import type { CoinQuantity, BN } from 'fuels';
+import { BaseAssetId, FUEL_NETWORK_URL, Provider, Wallet } from 'fuels';
+
+/**
+ * @group node
+ */
+describe(__filename, () => {
+  let provider: Provider;
+  let privateKey: string;
+  const assetId = BaseAssetId;
+
+  beforeAll(async () => {
+    provider = await Provider.create(FUEL_NETWORK_URL);
+    const wallet = await generateTestWallet(provider, [
+      [1000, BaseAssetId],
+      [1000, ASSET_A],
+    ]);
+    privateKey = wallet.privateKey;
+  });
+
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+
+  /* eslint-disable no-console */
+  it('should fetch specific balance just fine', async () => {
+    vi.spyOn(console, 'log').mockImplementation(() => ({}));
+
+    // #region checking-balances-1
+    const myWallet = Wallet.fromPrivateKey(privateKey, provider);
+
+    const balance: BN = await myWallet.getBalance(assetId);
+
+    // Amount is returned as a BigNumber
+    console.log(`Balance amount: ${balance.toString()}`);
```

Comment:
```suggestion
    // balance: 0x...
```

We don't usually use console in snippets to demo values?

Reply:
The logs were being suppressed but I've removed them